packages/feedback/src/FeedbackForm.js

```diff
@@ -163,16 +170,20 @@ const FeedbackForm = ({
                   options={aboutOptions}
                 />
               )}
-              <Field
-                type="textarea"
-                name="feedback"
-                placeholder={
-                  (active && active.placeholder) ||
-                  'Feedback? Requests? Defects?'
-                }
-                style={{ resize: 'none' }}
-                rows="2"
-              />
+              <Label for="feedback_input">
+                {(active && active.label) || 'Feedback? Requests? Defects?'}
+                <Field
+                  type="textarea"
+                  name="feedback"
+                  placeholder={
+                    (active && active.placeholder) ||
+                    'Feedback? Requests? Defects?'
+                  }
+                  style={{ resize: 'none' }}
+                  rows="2"
+                  id="feedback_input"
+                />
+              </Label>
```

Comment:
I think you should move this closing tag `</Label>` to 175 so the `Field` is not a child of `Label`

Reply:
The Field is a child of the Label so that when someone uses a screen reader for the Field they can see the label text is linked to the field. Want me to use aria-labelby instead?

Reply:
Yes, I think that would be better

Reply:
updated

Reply:
Do you need the aria-labelledby? Since you have "for" then the label and input should already be programmatically linked (the point of wrapping the label around the field was so you could skip explicitly assigning the "for" attribute and instead have it implicitly assigned, but I know that's just preference for some). iirc aria-labelledby is more so for linking elements without an explicit label, or in cases where the for attribute is difficult to apply within the label. 

Side note: doesn't this also pull from @availity/form? I think field has a label prop that you can pass in to skip this altogether.